### PR TITLE
De-template some internal methods of EtcdConsistentStore.

### PR DESCRIPTION
**Pull request by pphaneuf**

### CODE REVIEW

**Comment by pphaneuf** [comment]
After quite a bit of struggle with flaky builds, Travis is finally green. 😛

**Comment by eranmes** [comment]
Quick! Commit before it's red again!

File: cpp/log/consistent_store.h [hunk 1]
```diff
@@ -79,6 +81,10 @@ class EntryHandle : public EntryHandleBase {
     return &entry_;
   }
 
+  bool SerializeToString(std::string* output) const override {
```
Comment (eranmes) [inline cpp/log/consistent_store.h]: One may wonder why this method is needed given you could simply call Entry().SerializeToString ?

Comment (pphaneuf) [inline cpp/log/consistent_store.h]: This is to implement the pure virtual in the parent class, which allows method that only need to write the entry to etcd (without needing anything else from the entry) to take the non-templated parent class (avoiding the need to be templated themselves).

**Pull request merged**